assembly/build.gradle

```diff
@@ -1,121 +1,43 @@
-apply plugin: 'com.github.johnrengelman.shadow'
 apply plugin: 'java'
 
 description = "Sparkling Water Assembly"
 
 dependencies {
-    // Sparkling Water Core
+    // Sparkling Water libraries
+    compile project(":sparkling-water-ml")
+    compile project(":sparkling-water-repl")
     compile(project(":sparkling-water-core")) {
         exclude group: "javax.servlet", module: "servlet-api"
     }
-
-    compile(project(":sparkling-water-examples"))
-
-    compile(project(":sparkling-water-ml"))
-
     compile(project(":sparkling-water-extension-stack-trace"))
 
-    // And Scala library
-    compile "org.scala-lang:scala-library:${scalaVersion}"
-
+    // Spark libraries
+    compile "org.apache.spark:spark-core_${scalaBaseVersion}:${sparkVersion}"
+    compile "org.apache.spark:spark-sql_${scalaBaseVersion}:${sparkVersion}"
+    compile "org.apache.spark:spark-mllib_${scalaBaseVersion}:${sparkVersion}"
+    compile "org.apache.spark:spark-repl_${scalaBaseVersion}:${sparkVersion}"
 }
 
 jar {
     enabled = false // we do not need to generate jar file in this case
     archiveBaseName = "${jar.archiveBaseName.get()}_${scalaBaseVersion}"
 }
 
+apply plugin: 'com.github.johnrengelman.shadow'
```

Comment:
I personally prefer to have plugin specifications at the top of the file

Reply:
No problem :-)